src/Mobility/index.js

```diff
@@ -189,21 +209,34 @@ export default () => {
     selectedDate,
   ]);
 
+  // Zoom to selected country, if the selected country has bounds defined.
+  useEffect(() => {
+    if(!mapInitNeeded) {
+      if(!!config.defaults.countries[selectedCountryId].bounds) {
+        mapElement.current._map.fitBounds(config.defaults.countries[selectedCountryId].bounds, {
+          padding: 20
+        });
+      }
+    }
+  }, [selectedCountryId]);
+
   const mapInit = (map) => {
     if (mapInitNeeded) {
       setMapInitNeeded(false);
     }
   };
 
   const popup = popupEnabled ? (
+    popupDetails.feature && !!mobilityData[popupDetails.feature.id.toString()]
```

Comment:
Not a big deal but it looks like you could string these conditions together with `&&` and avoid the ternaries and the `: null : null` completely since we're taking the inner positive case only.

Reply:
good call, will fix!